.gitignore

```diff
@@ -6,3 +6,4 @@
 e2e_report.html
 bin
 out
+d2
```

Comment:
why ignore these? we want to use it for examples/docs, tests from files

Reply:
that's the d2 binary being ignored if you run `go build`, not `*.d2`.

Reply:
oh lol my b